app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -9,8 +9,8 @@
     <EditText
         android:id="@+id/etUsername"
         android:layout_width="match_parent"
-        android:layout_height="48dp"
-        android:layout_marginTop="250dp"
+        android:layout_height="@dimen/clickable"
```

Comment:
Should we be using the same dimension values as in `activity_login`? Can we use the variable name to describe the use case if so?